lib/ocpp/src/ocpp.c

```diff
@@ -0,0 +1,747 @@
+/*
+ * Copyright (c) 2024 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ocpp.h"
+#include "ocpp_i.h"
+#include <zephyr/posix/time.h>
+
+LOG_MODULE_REGISTER(ocpp, LOG_LEVEL_INF);
```

Comment:
You should allow user to set the log level via Kconfig, see examples in other network libraries